src/DependencyInjection/DI/src/DependencyInjectionEventSource.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.DependencyInjection.ServiceLookup;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    [EventSource(Name = "Microsoft-Extensions-DependencyInjection")]
+    internal sealed class DependencyInjectionEventSource : EventSource
+    {
+        public static readonly DependencyInjectionEventSource Instance = new DependencyInjectionEventSource();
```

Comment:
nit: The idiomatic name of this is `Log`... not sure why, but it just is :)

https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventsource?view=netframework-4.7.2#examples

Not sure what other ASP.NET Core sources use though.

Reply:
Renamed.